testvectors_v1/mldsa_65_standard_verify_test.json

```diff
@@ -0,0 +1,1026 @@
+          "result": "invalid",
+          "flags": [
+            "InvalidHintsEncoding"
+          ]
+        },
+        {
+          "tcId": 20,
+          "comment": "signature with omega+1 hints (causing a buffer overflow)",
```

Comment:
Not sure I get the point. The last byte of the signature is 0x37 (=55) which is exactly omega for ML-DSA-65. Why are you saying there are omega + 1 hints?

Reply:
Hi,

Agreed with the analysis of @wangweij : there is no overflow during bit unpacking here and the `HinBitUnpack` should succeed. However, the result of the verification is still `invalid` (for another reason to be determined) , hence the fact that this vector still "passes" the various checks of people using it.

Regards,